### DISCUSSION TITLE
Suggested shortcuts uses incorrect case for account names

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#49595] [Mobile] [answered]
GitHub Mobile beta

1. Go to the home screen
2. Trigger the add shortcuts option

![Screenshot_20230309-015001.png](https://user-images.githubusercontent.com/2119212/223943151-cb886fe9-e95b-432c-9455-d3eeee961702.png)

Note that my user account is `jsoref` written entirely in lowercase letters. 
Note that the suggestions include a bunch where the `j` is transformed into uppercase without justification.

**Comment by stevepopovich** [comment, marked as answer]
Hi @jsoref, I filed an issue. Thanks!
